x/mongo/driver/operation.go

```diff
@@ -957,7 +993,7 @@ func (op Operation) createQueryWireMessage(dst []byte, desc description.Selected
 	return bsoncore.UpdateLength(dst, wmindex, int32(len(dst[wmindex:]))), info, nil
 }
 
-func (op Operation) createMsgWireMessage(ctx context.Context, dst []byte, desc description.SelectedServer,
+func (op Operation) createMsgWireMessage(ctx context.Context, maxTimeMS *int64, dst []byte, desc description.SelectedServer,
```

Comment:
Same as above: There's no valid use case for a non-positive `maxTimeMS` value, so we should change the `maxTimeMS` param type to a uint64 and only include it in the request if it's positive.

Reply:
Done.